.eslintrc.js

```diff
@@ -64,13 +64,6 @@ module.exports = {
     ],
     'no-use-before-define': 'off',
     'react/jsx-props-no-spreading': 0,
-    'sort-keys': [
```

Comment:
this rule was making development slower and I don't think that sorting keys in alphabetical order is all that important